### DISCUSSION TITLE
markdown解析器用的是什么？

### DISCUSSION THREAD

**Discussion by aswirl4** [halo-dev/halo#2744] [Q&A]
请问halo用的markdown解析器是什么？
看文档说用的是markdown-it这个markdown渲染器。
我想要一个能支持脚注footnote的语法的解析器， 不知道markdown-it是否也负责解析markdown语法。

**Comment by ruibaby** [comment]
现在是支持脚注的，你是用的哪个版本？

**Reply by aswirl4** [reply]
不好意思没说明清楚。
我是自己想要fork一份markdown解析器，在别处使用。

**Reply by ruibaby** [reply]
https://github.com/halo-dev/js-sdk/tree/master/packages/markdown-renderer
